edx-platform/pearson-certiport-theme/lms/templates/courseware/tabs.html

```diff
@@ -1,34 +0,0 @@
-## mako
```

Comment:
Why this module is in certprep and not here? Could you please review that the modules in certprep are correct?

Reply:
@anfbermudezme tabs.html has to be removed in CertPREP theme since it doesn't contain custom changes.